Makefile.common

```diff
@@ -201,7 +201,13 @@ common-unused:
 .PHONY: common-build
 common-build: promu
 	@echo ">> building binaries"
-	$(PROMU) build --prefix $(PREFIX) $(PROMU_BINARIES)
```

Comment:
Makefile.common is automatically synced from https://github.com/prometheus/prometheus/blob/main/Makefile.common so this would need to change there

Reply:
ACK, opened https://github.com/prometheus/prometheus/pull/15712.